modules/doc/content/newsletter/2024/2024_06.md

```diff
@@ -1,14 +1,24 @@
 # MOOSE Newsletter (June 2024)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in July 2024
-for a complete description of all MOOSE changes.
-!alert-end!
-
 ## MOOSE Improvements
 
+
+### Using Physics on Components
+
+A new component, [FileMeshPhysicsComponent.md], was created in the thermal hydraulics module to leverage the [Physics](Physics/index.md) system. This facilitates the use of multi-dimensional equations in a thermal-hydraulics simulations, including notably the finite volume computational fluid dynamics capabilities of the Navier Stokes module.
+
+### Facilitating the use of mixture models
+
+The [WCNSFVTwoPhaseMixturePhysics.md] was added to the Navier Stokes module. This facilitates the set up of fully coupled finite volume simulations using this simple two phase mixture model. It can be used for both solidification and liquid mixture problems.
```

Comment:
```suggestion
The [WCNSFVTwoPhaseMixturePhysics.md] was added to the Navier Stokes module. This facilitates the setup of fully coupled finite volume simulations using a simple two phase mixture model. It can be used for both solidification and liquid mixture problems.
```